utils/config/tokenrefresh.go

```diff
@@ -153,13 +203,14 @@ func createTokensForConfig(serverDetails *ServerDetails, expirySeconds int) (ser
 
 	newToken, err := servicesManager.CreateToken(createTokenParams)
 	if err != nil {
-		return services.CreateTokenResponseData{}, err
+		return auth.CreateTokenResponseData{}, err
 	}
 	return newToken, nil
 }
 
 func CreateInitialRefreshableTokensIfNeeded(serverDetails *ServerDetails) (err error) {
-	if !(serverDetails.TokenRefreshInterval > 0 && serverDetails.RefreshToken == "" && serverDetails.AccessToken == "") {
+	if !(serverDetails.ArtifactoryTokenRefreshInterval > 0 && serverDetails.RefreshToken == "" && serverDetails.AccessToken == "") ||
```

Comment:
Seems to me like there's a mix up between the tokens, and I'm not sure why the second condition is needed. Care to explain?

Reply:
Yes, I got confused between the tokens - fixed it.